sqlparse/visitor.go

```diff
@@ -62,6 +62,8 @@ func (v *astVisitor) leaveImpl(n ast.Node) error {
 				v.exprStack[0],
 				v.exprStack[1:]...,
 			)
+		default:
```

Comment:
Is it correct to have this as the `default` case? I guess previously we just returned all columns, so we do want to do a Project in all cases, is that right?

Reply:
I think we'd want to extend this to support something like `select *` in the future, but I think for what we have now this is okay?